src/rviz/image/image_display_base.h

```diff
@@ -77,9 +79,15 @@ protected Q_SLOTS:
   /** @brief Update queue size of tf filter  */
   virtual void updateQueueSize();
 
+  /** @brief Prepare for or stop resetting when timed out */
+  virtual void updateResetTO();
```

Comment:
Why is this virtual? Do you see any need to overwrite that method in sub classes?